2025-04-25 14:13:17,026 - WARNING - [_log] -  * Debugger is active!
2025-04-25 14:13:17,031 - INFO - [_log] -  * Debugger PIN: 101-233-949
2025-04-25 14:13:17,044 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:17] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:17,138 - WARNING - [_log] -  * Debugger is active!
2025-04-25 14:13:17,138 - INFO - [_log] -  * Debugger PIN: 107-410-311
2025-04-25 14:13:17,959 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:17] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:18,460 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:18] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:18,637 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:18] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:18,821 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:18] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:19,012 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:19] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:19,196 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:19] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:19,388 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:19] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:19,572 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:19] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:19,748 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:19] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:19,917 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:19] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:20,105 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:20] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:20,298 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:20] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:20,461 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:20] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:20,654 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:20] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:20,845 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:20] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:21,036 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:21] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:21,221 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:21] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:21,409 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:21] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:21,611 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:21] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:22,091 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:22] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:22,295 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:22] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:22,481 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:22] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:22,678 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:22] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:13:22,845 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:13:22] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:14:28,673 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:14:28] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:14:30,383 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:14:30] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:14:30,815 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:14:30] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:14:31,002 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:14:31] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:14:31,184 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:14:31] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:14:31,381 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:14:31] "[33mPOST /initiate-upload HTTP/1.1[0m" 404 -
2025-04-25 14:15:23,699 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:23] "GET /?vscodeBrowserReqId=1745570723674 HTTP/1.1" 200 -
2025-04-25 14:15:23,750 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:23] "[33mGET /static/style.css HTTP/1.1[0m" 404 -
2025-04-25 14:15:32,506 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:32] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:32,539 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:32] "GET /stream-progress/4424e873-5a52-4d0b-aceb-665ee3a72c93 HTTP/1.1" 200 -
2025-04-25 14:15:32,552 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:34,525 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:34] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:34,553 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:34] "GET /stream-progress/77c2fc92-ac32-4e0e-9fc7-80526ed12f04 HTTP/1.1" 200 -
2025-04-25 14:15:34,572 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:35,574 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:35] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:35,599 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:35] "GET /stream-progress/141480ac-d890-4d15-bb34-8cc7f9e19912 HTTP/1.1" 200 -
2025-04-25 14:15:35,626 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:36,149 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:36] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:36,181 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:36] "GET /stream-progress/2ed54bf1-7ded-4c11-bb29-1150ac629178 HTTP/1.1" 200 -
2025-04-25 14:15:36,195 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:36,594 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:36] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:36,622 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:36] "GET /stream-progress/e896780e-6b97-4112-a67c-8b6ec997390e HTTP/1.1" 200 -
2025-04-25 14:15:36,646 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:37,065 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:37,098 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "GET /stream-progress/b0dabb9e-7cca-40d7-a90e-b457fd7fa6f0 HTTP/1.1" 200 -
2025-04-25 14:15:37,114 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:37,286 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:37,312 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "GET /stream-progress/4bb3251b-4c70-4bad-87b9-a3b69d6d4a68 HTTP/1.1" 200 -
2025-04-25 14:15:37,329 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:37,733 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:37,748 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "GET /stream-progress/4e501343-16f0-4cd9-984d-6837e15d3abd HTTP/1.1" 200 -
2025-04-25 14:15:37,782 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:37,957 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:37,986 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:37] "GET /stream-progress/1edbe893-88f9-43cf-96bd-6d6bd7604667 HTTP/1.1" 200 -
2025-04-25 14:15:38,002 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:38,154 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:38,171 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "GET /stream-progress/621f9290-f42a-4efa-a250-a6bb3fc98016 HTTP/1.1" 200 -
2025-04-25 14:15:38,202 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:38,356 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:38,385 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "GET /stream-progress/cdd25371-b98a-4f42-8545-b419395f5c91 HTTP/1.1" 200 -
2025-04-25 14:15:38,402 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:38,553 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:38,572 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "GET /stream-progress/9d24e5df-cb32-4124-a19c-cc9530a74226 HTTP/1.1" 200 -
2025-04-25 14:15:38,603 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:38,761 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:38,787 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "GET /stream-progress/32030702-4820-4aa1-b2dd-d174ca1c38d5 HTTP/1.1" 200 -
2025-04-25 14:15:38,801 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:38,988 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:38] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:39,021 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:39] "GET /stream-progress/fd36746a-5676-4015-a090-f4dcb6212b35 HTTP/1.1" 200 -
2025-04-25 14:15:39,039 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:15:39,421 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:39] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:15:39,444 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:15:39] "GET /stream-progress/698f4e72-2e73-4e44-a58d-50d0b3f8a1f2 HTTP/1.1" 200 -
2025-04-25 14:15:39,473 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:12,299 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:12] "GET / HTTP/1.1" 200 -
2025-04-25 14:16:12,407 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:12] "[33mGET /static/style.css HTTP/1.1[0m" 404 -
2025-04-25 14:16:17,485 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:17] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:17,632 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:17] "GET /stream-progress/1ab42c3e-56de-4092-a802-ad7d61c550ca HTTP/1.1" 200 -
2025-04-25 14:16:17,659 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:18,587 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:18] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:18,675 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:18] "GET /stream-progress/ef8b4c88-9e3f-41d3-a875-16bf725e3753 HTTP/1.1" 200 -
2025-04-25 14:16:18,697 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:19,255 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:19] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:19,362 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:19] "GET /stream-progress/5ffb0994-0f85-4446-bc45-335e03155db9 HTTP/1.1" 200 -
2025-04-25 14:16:19,377 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:19,880 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:19] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:19,969 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:19] "GET /stream-progress/cb67e05e-dc26-4761-b7f4-84cf5d04d8b5 HTTP/1.1" 200 -
2025-04-25 14:16:19,985 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:20,514 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:20] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:20,643 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:20] "GET /stream-progress/5ca2d783-90b4-487c-8cd8-c52a44deb854 HTTP/1.1" 200 -
2025-04-25 14:16:20,665 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:21,074 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:21] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:21,210 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:21] "GET /stream-progress/2682affe-b3bc-4f35-9720-f1dd9d8dbc0a HTTP/1.1" 200 -
2025-04-25 14:16:21,235 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:24,349 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:24] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:24,778 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:24] "GET /stream-progress/38b33ce2-3121-419b-ac1f-6221e5d4c40d HTTP/1.1" 200 -
2025-04-25 14:16:24,800 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:25,725 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:25] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:26,101 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:26] "GET /stream-progress/eb76299a-fba4-4e93-9817-7a89203df336 HTTP/1.1" 200 -
2025-04-25 14:16:26,111 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:27,054 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:27] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:27,438 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:27] "GET /stream-progress/d77744a0-ca08-47cf-98c2-ec7656e5b353 HTTP/1.1" 200 -
2025-04-25 14:16:27,455 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:16:28,224 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:28] "POST /upload HTTP/1.1" 200 -
2025-04-25 14:16:28,642 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:16:28] "GET /stream-progress/d3dad185-b3b1-4091-a014-c606172fe82a HTTP/1.1" 200 -
2025-04-25 14:16:28,659 - ERROR - [_log] - Error on request:
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 370, in run_wsgi
    execute(self.server.app)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 334, in execute
    write(data)
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\serving.py", line 302, in write
    assert isinstance(data, bytes), "applications must write bytes"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: applications must write bytes
2025-04-25 14:18:40,599 - INFO - [_log] - 127.0.0.1 - - [25/Apr/2025 14:18:40] "[33mGET /static/style.css HTTP/1.1[0m" 404 -
2025-04-25 14:32:02,633 - INFO - [_log] -  * Detected change in 'E:\\Telegram_tool\\Telegram_Bot_Tool\\Telegram_app.py', reloading
